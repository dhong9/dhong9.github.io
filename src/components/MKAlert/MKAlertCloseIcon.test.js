import React from 'react';
import renderer from 'react-test-renderer';
import MKAlertCloseIcon from 'components/MKAlert/MKAlertCloseIcon';

// Material Kit 2 React themes
import { ThemeProvider } from '@mui/material/styles';
import theme from "assets/theme";

describe('MKAlertCloseIcon', () => {
    it('renders', () => {
        const component = renderer.create(
            <ThemeProvider theme={theme}>
                <MKAlertCloseIcon onClick={jest.fn()}>Hello!</MKAlertCloseIcon>
            </ThemeProvider>
        );

        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
