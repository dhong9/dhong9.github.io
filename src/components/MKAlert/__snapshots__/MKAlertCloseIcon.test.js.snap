// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MKAlertCloseIcon renders 1`] = `
<span
  className="css-mqrg1u"
  onClick={[MockFunction]}
>
  Hello!
</span>
`;
